web/src/components/HubListDateSelector/HubListDateSelector.tsx

```diff
@@ -0,0 +1,26 @@
+// import { useState } from 'react'
+import DaySelectionRow from '../bit.dev/day-selection-row'
+import PublicHolidayToggleSwitch from '../bit.dev/public-holiday-toggle'
+
+const HubListDateSelector = () => {
+  const daysOfWeek = [
```

Comment:
I'll move these into a presets.ts file and import these that way we can separate data from components

Reply:
Done

Reply:
nice, well-done